utilities/rpp-performancetests/HIP_NEW/Tensor_hip_pln1.cpp

```diff
@@ -689,6 +697,108 @@ int main(int argc, char **argv)
 
             break;
         }
+        case 40:
+        {
+            test_case_name = "erode";
+
+            Rpp32u kernelSize = additionalParam;
+            for (i = 0; i < images; i++)
```

Comment:
Modified comment for better clarity on the use case of the ROI override.

Reply:
please put the for loop itself into the comment section as well. use /* */ to block the entire block instead of using // to comment on each line. so something like below:
// Uncomment to run test case with an xywhROI override
/* for (i = 0; i < images; i++)
...
...


}*/